src/common/wincmn.cpp

```diff
@@ -3955,7 +3955,7 @@ wxAccStatus wxWindowAccessible::GetParent(wxAccessible** parent)
         if (*parent)
             return wxACC_OK;
         else
-            return wxACC_FAIL;
+            return wxACC_NOT_IMPLEMENTED;
```

Comment:
Is this related to the other changes? If not, it should be ideally extracted into a separate commit, with the explanation of why is this necessary in its commit message.